README

```diff
@@ -1,15 +1,20 @@
-Btrfs balancer is a set of scripts that create systemd service
-to periodically run btrfs balance operation on a devices filesystem.
+btrfs-balancer is a D-Bus service for running btrfs operations on
+a device's filesystem periodically via systemd, or on request.
 
 It consists of roughly 2 parts:
 
-* btrfs-balance script that does simple filesystem status check to see if
-  the balance operation is worth running and executes btrfs balance when needed.
-  The btrfs-balance script can be run also stand-alone.
-* systemd service files to run btrfs-balance periodically
+* btrfs-balancer D-Bus service with auto-activation that starts automatically
+  when requested and stops when idle.
+  The btrfs-balancer has a client mode in which it acts as a command line
+  client. Invoke "btrfs-balancer --help" for details.
+
+* systemd service files to run the btrfs-balancer maintenance method
+  periodically.
 
 Install from source:
-make install [DESTDIR=myrootfolder]
+  qmake
+  make
+  make install [DESTDIR=myrootfolder]
```

Comment:
I think that should be `make install [INSTALL_ROOT=myrootfolder]` (see [this blog post](https://davmac.wordpress.com/2007/02/21/qts-qmake/)), but might as well just remove the "install from source" instructions altogether.